.github/workflows/ci-guardrailpolicies.yml

```
@@ -0,0 +1,31 @@
+name: ci-guardrailpolicies
+
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
+  pull_request:
+
+permissions:
+  contents: read
+
+jobs:
+  opa-guardrail-policies-check:
+    name: opa-guardrail-policies-check
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+
+    - name: install gator cli & opa binaries
+      run: |
+        go install github.com/open-policy-agent/gatekeeper/v3/cmd/gator@master
+        curl -L -o opa https://openpolicyagent.org/downloads/latest/opa_linux_amd64
+        chmod 755 opa; cp opa /usr/local/bin/
+
+    - name: run opa & guardrails policies test.sh
+      run: |
+        export PATH=$PATH:/home/runner/go/bin; cd ${GITHUB_WORKSPACE}/pkg/operator/controllers/guardrails/policies; bash ./scripts/test.sh
```

Is there any reason you put the commands into one line?
Otherwise I would rather split it like `install gator cli & opa binaries` step.

---

Thanks for taking at the PR @bitoku, no reason for adding the commands in one line, I'll just split into two steps :+1:

---

I might have confused you, sorry.
I'm suggesting changing like this.

```suggestion
        export PATH=$PATH:/home/runner/go/bin
        cd ${GITHUB_WORKSPACE}/pkg/operator/controllers/guardrails/policies
        bash ./scripts/test.sh
```